Remotion/Web/Core/Res/Themes/NovaViso/HTML/DropDownMenu.css

```diff
@@ -1,374 +1,207 @@
-  background-size: 32px 15px;
-}
-*/
 span.DropDownMenuSelect a.DropDownMenuButton img
 {
-  min-height: 12px;
-  height: 90%;
-  height: calc(100% - 4px);
-  margin-top: 2px;
-  margin-bottom: 2px;
-  width: 12px;
-  margin-left: 2px;
-  margin-right: 2px;
+  width: 1.25em;
   background-image: url('../Image/sprite.svg#DropDownMenuButton');
-  background-size: 32px 15px;
   background-repeat: no-repeat;
 }
 
-span.DropDownMenuSelect a.DropDownMenuButton:focus img
-{
-  outline: dotted black 1px;
-  outline: auto 5px -webkit-focus-ring-color;
-}
-
-span.DropDownMenuSelect.disabled a.DropDownMenuButton img
-{
-  opacity: 0.4;
-  filter: progid:DXImageTransform.Microsoft.BasicImage(opacity=0.40),
-          progid:DXImageTransform.Microsoft.BasicImage(grayscale=1);
-}
-
-span.DropDownMenuSelect img
-{
-  border: 0;
-}
-
-div.DropDownMenuOptions
-{
-  z-index: 2147483647;
-  position: absolute;
-  right: 0;
-  margin: 0;
-  padding: 0;
-}
-
-ul.DropDownMenuOptions
+span.DropDownMenuSelect a.DropDownMenuLabel img
 {
-  border: solid 1px #AAAAAA;
-  left: 0;
-  right: 0;
-  margin: 0;
-  padding: 0;
-  list-style: none;
-  background-color: White;
-  background-image: url('../Image/sprite.svg#DropDownMenuItemIconPane');
-  background-size: 22px 1px;
-  background-repeat: repeat-y;
-  background-position: top left;
-}
+  padding-right: var(--remotion-themed-padding);
 
-ul.DropDownMenuOptions li a
-{
-  cursor: hand;
-  cursor: pointer;
-  text-decoration: none;
-  display: block;
-  margin: 1px;
-  padding: 1px 3px 3px 1px;
-  white-space: nowrap;
-  color: Black;
-  outline: none;
+  background-repeat: no-repeat;
 }
 
 ul.DropDownMenuOptions li.selected a
 {
   color: Black;
-  border: solid 1px #7CB2E6;
-  background-color: #E5F0FC;
-  padding: 0 2px 2px 0;
-}
-
-ul.DropDownMenuOptions li.selected a:active,
-ul.DropDownMenuOptions li a:active
-{
-  color: Black;
-  border: solid 1px #569CE0;
-  background-color: #C5E0FC;
-  padding: 0 2px 2px 0;
-}
-
-/* Workaround for IE-issue with a span inside an anchor tag blocking the :active pseudo class. */
-body.msie ul.DropDownMenuOptions li a
-{
-  position: relative;
-}
-
-/* Workaround for IE-issue with a span inside an anchor tag blocking the :active pseudo class. */
-body.msie ul.DropDownMenuOptions li a::before
-{
-  content: "";
-  position: absolute;
-  left: 0;
-  right: 0;
-  top: 0;
-  bottom: 0;
-}
-
-ul.DropDownMenuOptions li a img
-{
-  border: 0;
-  margin-right: 15px;
-  vertical-align: middle;
-}
-
-ul.DropDownMenuOptions li a span
-{
-  display: inline-block;
-  padding: 2px 0px;
+  background-color: var(--color-highlight);
+  border-radius: var(--remotion-themed-border-radius);
 }
 
 ul.DropDownMenuOptions li.DropDownMenuItemDisabled a
 {
+  display: flex;
   cursor: default;
   border-style: none;
-  padding: 1px 3px 3px 3px;
-  color: gray;
-}
-
-ul.DropDownMenuOptions li.DropDownMenuItemDisabled.selected a
-{
-  background-color: #ECEEEC;
+  color: var(--color-contrast-separator);
```

Comment:
please check all disabled items, we have dediacated colors for them.

Reply:
I haven't checked where exactly, but there's a gray-hover effect for diabled items. Disabled items do not interact.